py3status/module.py

```diff
@@ -102,7 +107,17 @@ def load_methods(self, module, user_modules):
             # apply module configuration from i3status config
             mod_config = self.i3status_thread.config.get(module, {})
             for config, value in mod_config.items():
-                setattr(self.module_class, config, value)
+                # names starting with '.' are private
```

Comment:
Being linuxy works here too I think.
On Mar 17, 2016 02:38, "Ultrabug" <notifications@github.com> wrote:

> In py3status/module.py
> <https://github.com/ultrabug/py3status/pull/203#discussion_r56465335>:
>
> > @@ -102,7 +107,17 @@ def load_methods(self, module, user_modules):
> >              # apply module configuration from i3status config
> >              mod_config = self.i3status_thread.config.get(module, {})
> >              for config, value in mod_config.items():
> > -                setattr(self.module_class, config, value)
> > +                # names starting with '.' are private
>
> wouldn't _ or __ be a more pythonic way of defining this private state ?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/ultrabug/py3status/pull/203/files/3ea249bd09d2e1780133a3149db7b3bf97faefab#r56465335>
>